radixdlt/src/test/java/com/radixdlt/EpochsConsensusModuleTest.java

```diff
@@ -93,13 +97,32 @@ protected void configure() {
 				bind(VerifiedLedgerHeaderAndProof.class).toInstance(mock(VerifiedLedgerHeaderAndProof.class));
 				bind(Integer.class).annotatedWith(BFTSyncPatienceMillis.class).toInstance(200);
 			}
+
+
+			@Provides
+			@PacemakerTimeout
+			long pacemakerTimeout() {
```

Comment:
As noted elsewhere would be better and more consistent if bound in configure like BFTSyncPatienceMillis